gui/syncthing/core/eventService.js

```diff
@@ -18,14 +18,43 @@ angular.module('syncthing.core')
                 return;
             }
             $rootScope.$broadcast(self.ONLINE);
-
+            //GREEN
+            $("link[rel='shortcut icon']").attr("href","assets/img/favicon_idle.png")
+            
             if (lastID > 0) {   // not emit events from first response
+                var green_counter=0;
+                var red_counter=0;
+                
                 data.forEach(function (event) {
                     if (debugEvents) {
                         console.log("event", event.id, event.type, event.data);
                     }
                     $rootScope.$broadcast(event.type, event);
+                    
+                    if (event.type=='FolderErrors') {
```

Comment:
You should use the enum values, or define enum values if they are not defined. Same like self.OFFLINE, etc.